reactor-adapter/src/main/kotlin/reactor/adapter/rxjava/RxJava2AdapterExt.kt

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright (c) 2018-2019 VMware Inc. or its affiliates, All Rights Reserved.
```

Comment:
Wrong end year, or we do not aim for 2021 everywhere?

Reply:
right, I should have gone the last mile here and updated the end of the range. will quickly do that